src/base/bittorrent/session.cpp

```diff
@@ -2417,6 +2422,16 @@ void Session::configureListeningInterface()
     configureDeferred();
 }
 
+int Session::globalDownloadRate() const
```

Comment:
Don't we overuse "global" in `Session` member names? Are there other download/upload rates so we can't just use `downloadRate`/`uploadRate`?

Reply:
one could even argue that these *aren't* global. they are the *session* transfer rates. Presumably, if you'd have two `Session` objects, they could return different values here

Reply:
these names were based on other Session members `globalUploadSpeedLimit` and `globalDownloadSpeedLimit`, do I still need to change these names (globalDownloadRate -> downloadRate and globalUploadRate -> uploadRate)?

Reply:
I'm not a stake-holder/approver in this repo, I'm just commenting.

Reply:
IMO, all per-session indicators should have no "global" prefixes (e.g. `Session::uploadRate()` clearly shows that it's upload rate of all session torrents).